src/components/footer/_macro.spec.js

```diff
@@ -478,6 +478,22 @@ describe('macro: footer', () => {
           expect($('.custom-logo').length).toBe(1);
         });
       });
+
+      it('it has the additional logo image and link', () => {
```

Comment:
```suggestion
      it('has the additional logo image and link', () => {
```
This change makes it conform with the way other tests are written